### ISSUE TITLE
Social Media Icons are not being fetched

### ISSUE THREAD

**Issue by SumanthNadiger** [rahuldkjain/github-profile-readme-generator#385]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

Join the **Discord Server** for further discussions.

<a href="https://discord.gg/HHMs7Eg">
<img src="https://discord.com/assets/e4923594e694a21542a489471ecffa50.svg" alt="GPRG Discord Server Link" width="300px"/>
</a>

Server Link: https://discord.gg/HHMs7Eg

**Comment by Shrestha7** [comment]
Yes, Social Media Icons are not working for some reason.
This is what it showing after the generating profile .
![image](https://user-images.githubusercontent.com/40354116/116875735-1a58b080-ac3b-11eb-850d-1ad8e0111f6e.png)

**Comment by its-Kumar** [comment]
Same for me, I tried to open the location of icon, and it says **404 Not found**

**Comment by its-Kumar** [comment]
I manually copied the location of the icon and found this
```diff
- https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/neutral-icons/src/images/icons/Social/hackerearth.svg    # automatically generated link
+ https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/6253936f99716cd30c07055d5d10e9332af37171/src/images/icons/Social/hackerearth.svg    # actual link 
```

**Comment by Shrestha7** [comment]
@its-Kumar  How did you get the actual link and will that works?

**Comment by its-Kumar** [comment]
@Shrestha7 yes, it works. I get it from the github repo from manually surfing the folders

**Comment by rahuldkjain** [comment]
@Shrestha7 would you like to fix the issue?

**Comment by Shrestha7** [comment]
> @Shrestha7 would you like to fix the issue?

@rahuldkjain  ya sure

**Comment by SumanthNadiger** [comment]
I forked the original post and copied the original location of the .svg file it worked

**Comment by SumanthNadiger** [comment]
I think profile generator is generating relative path not absolute path to the file. If that helps

**Comment by its-Kumar** [comment]
> I think profile generator is generating relative path not absolute path to the file. If that helps

Yes you're right @SumanthNadiger

**Comment by Shrestha7** [comment]
@rahuldkjain Please check my pull request.

**Comment by MominBinShahid** [comment]
@rahuldkjain we can close this as this is fixed here #387
Thanks to @Shrestha7

**Comment by rahuldkjain** [comment]
It's fixed now.
